WorkloadManagementSystem/DB/JobDB.py

```diff
@@ -66,7 +66,7 @@
 #############################################################################
 
 JOB_STATES = ['Submitting', 'Received', 'Checking', 'Staging', 'Waiting', 'Matched',
-              'Running', 'Stalled', 'Done', 'Completed', 'Failed']
+              'Running', 'Stalled', 'Completing', 'Done', 'Completed', 'Failed']
 JOB_FINAL_STATES = ['Done', 'Completed', 'Failed']
```

Comment:
I notice now that `Completed` is both in `JOB_STATES` and in `JOB_FINAL_STATES`. Maybe there's (or there was) a good reason, does it still stand?

Reply:
Well, Completed was already in FINAL_STATES although it was not a final state for jobs (but now these are indeed in principle the only statuses a job should have when terminating at a site), but my impression is that this is irrelevant ;-). 
It is only used for counting in the SiteSummaryWeb, therefore I think we can leave it like this...

Reply:
OK, leave it. 

Another thing: are updates to StalledJobAgent necessary or not?